portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase

```diff
@@ -32,13 +32,41 @@ definition {
 	}
 
 	@description = "Verify that a Paragraph Field can be created"
-	@ignore = "true"
 	@priority = "5"
 	test CanBeCreated {
 		property portal.acceptance = "false";
 
-		// TODO LPS-131221 CanBeCreated pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "1",
+			fieldType = "Paragraph");
+
+		var fieldParagraphField = FormFields.getFieldName();
+
+		Form.editFieldLabelMetal(fieldValue = "Paragraph");
+
+		FormFields.editParagraphBody(content = "Test");
```

Comment:
You are declaring a var here but it is not being referenced elsewhere. Is it possible to take this line out and have the test still pass?